src/EnergyPlus/HVACMultiSpeedHeatPump.cc

```diff
@@ -619,7 +546,7 @@ namespace HVACMultiSpeedHeatPump {
             if (lAlphaBlanks(2)) {
                 MSHeatPump(MSHPNum).AvaiSchedPtr = ScheduleManager::ScheduleAlwaysOn;
```

Comment:
`auto &thisMSHeatPump` would help here. Also there is an unnecessary `IsNameEmpty` above.